index.js

```diff
@@ -571,6 +582,25 @@ function issecure(req, trustProxy) {
 }
 
 /**
+ * Create a new Cookie for this request.
+ *
+ * @param {Object} cookieOptions
+ * @param {Object} req
+ * @param {Boolean} [trustProxy]
+ * @return {Cookie}
+ * @private
+ */
+function createCookie(cookieOptions, req, trustProxy) {
+  cookieOptions = cookieOptions || {};
```

Comment:
@dougwilson I don't think this is true, it appears to only be applicable when [mentioning arguments](https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#3-managing-arguments). If you have another source, I'd be curious to see it.